pyproject.toml

```diff
@@ -18,8 +18,7 @@ version = "0.1.0"
 description = "Utilities for use on EODHP"
```

Comment:
I think this will be v0.1.2

Reply:
I've added a change which should auto-generate the version based on Git tags rather than it having to be changed in this file. It gives `eodhp-utils-0.1.1.post15+git.b9549bf4` at the moment - 15 commits after tag 0.1.1 - but in theory will become 0.1.2 when there's a tag.